### ISSUE TITLE
lv2lint fail

### ISSUE THREAD

**Issue by mxmilkiib** [sjaehn/BChoppr#10]
```
...
==> Starting check()...
lv2lint 0.10.0
Copyright (c) 2016-2021 Hanspeter Portner (dev@open-music-kontrollers.ch)
Released under Artistic License 2.0 by Open Music Kontrollers
lilv_world_add_plugin(): warning: Duplicate plugin <https://www.jahnichen.de/plugins/lv2/BChoppr>
lilv_world_add_plugin(): warning: ... found in file:///usr/lib/lv2/BChoppr.lv2/
lilv_world_add_plugin(): warning: ... and      file:///home/milk/.cache/yay/bchoppr-git/src/bchoppr/BChoppr.lv2/ (ignored)
<https://www.jahnichen.de/plugins/lv2/BChoppr>
    [WARN]  Plugin Author Email
              foaf:mbox not found
              seeAlso: <http://lv2plug.in/ns/lv2core#project>
  <https://www.jahnichen.de/plugins/lv2/BChoppr#gui>
    [FAIL]  UI Symbols
              binary exports superfluous globally visible symbols: 
                * puglFreeView
                * puglSetBackend
                * puglGetClipboard
                * puglFreeWorld
                * puglGetTime
                * puglSetCursor
                * puglHideWindow
                * puglSetClassName
                * puglStubBackend
                * puglRequestAttention
                * ... there is more, but the rest is being truncated
              seeAlso: <http://lv2plug.in/ns/lv2core#binary>

==> ERROR: A failure occurred in check().
    Aborting...
error making: bchoppr-git
```

**Comment by sjaehn** [comment]
Thanks for reporting. I'll check it.

**Comment by sjaehn** [comment]
Confirmed. The bad guy is there: https://github.com/sjaehn/BChoppr/blob/master/src/BWidgets/pugl/pugl/pugl.h#L29 . I'll fix it later. And for all B.Music plugins.

**Comment by sjaehn** [comment]
Fixed in 8d9bed21b8d30494da6d9cc222d90ffe55ac51a2
